app/views/api/v2/realms/main.json.rabl

```diff
@@ -1,3 +1,5 @@
 object @realm
 extends "api/v2/realms/base"
+extends "api/v2/smart_proxies/children_nodes"
+
 attributes :name, :realm_proxy_id, :realm_type, :created_at, :updated_at
```

Comment:
Oh sorry - I hadn't meant to remove them. I meant to replace the explicit listings of `attributes :realm_proxy_id` by code that automatically calls `attributes` with `_id` and `_name` for each registered smart proxy association. IDs and names for new associations from plugins will automatically appear in the output too then, not just through full child nods.